config/system.php

```diff
@@ -350,6 +350,13 @@
 'default_value'=>'n'
 );
 
+$system__options_general[]=array(
+'option_name'=>'hide_planned_sessions_in_public_calendar',
```

Comment:
I think we need indents here for the readability. Otherwise looks good.

Reply:
The whole file does not have indentation for system options (to me it seems ok in terms of readability). Let's consider indentation for the whole file in a larger cleanup, rather than doing it only for this option now.